### ISSUE TITLE
Tests execution hags with the latest version of  Test Execution Command Line Tool Version (16.1.0)

### ISSUE THREAD

**Issue by tcoreal** [microsoft/vstest#2021]
## Description
Tests execution hags with the latest version of  Test Execution Command Line Tool Version (16.1.0) dotnet sdk 2.1.700
Previous version (16.0.1 dotnet sdk 2.1.604) with the same arguments works fine

## Steps to reproduce
command line ```dotnet test --logger "trx;LogFileName=TestResults.trx" UnitTests.csproj```

Docker Engine Version: 18.09.1
Kernel Version: Linux b8b2a23dda36 4.15.0-1035-aws #37-Ubuntu SMP Mon Mar 18 16:15:14 UTC 2019 x86_64 Linux
dotnet core container mcr.microsoft.com/dotnet/core/sdk:2.1.700

## Expected behavior
> Test not hang and run succesfully

## Actual behavior
>Tests hang for 22 min, print repeating message "Test run in progress.Test run in progress.Test run in progress.Test run in progress...." and after that tests successfully finish

**Comment by mgkeen** [comment]
I've been getting the same thing. However i'm getting it when running `dotnet test --filter Category=Integration` and it seems to be ignoring the filter altogether as well when on linux

**Comment by mgkeen** [comment]
Downgrading to `16.0.1` fixed both the hanging and the filtering issues for me. It wasn't obvious how to downgrade. As i'm only seeing the issue on my CI i just pinned the version of the container i was using to `mcr.microsoft.com/dotnet/core/sdk@sha256:d0a71e1312be2618f320a7b7cc6d7423af3810542b3ffd35438108800ecfb958` to solve the problem for now.

**Comment by tcoreal** [comment]
Yes, exactly, downgrading to ‘2.1.604’ tag fixed the problem, but this just underlines the problem with the current version

**Comment by brpratt** [comment]
This may be same issue as:

https://github.com/dotnet/core/issues/2775

There is an insightful comment by @JSkimming in the conversation:

https://github.com/dotnet/core/issues/2775#issuecomment-495083631

He's also created a PR to potentially fix this issue:

https://github.com/microsoft/vstest/pull/2024

**Comment by jayaranigarg** [comment]
@tcoreal @mgkeen : We have released [16.1.1 ](https://github.com/microsoft/vstest/releases/tag/v16.1.1)version of nuget package with the fix #2024 . Please try that out and let us know if you are still hitting the issue.

**Comment by JSkimming** [comment]
How do we try it out?

[Here's the build](https://circleci.com/gh/JSkimming/abioc/1031) with the updated package and it still takes ages, also `dotnet test` still reports the following as still using `16.1.0`.

```
Test run for /root/project/test/Abioc.Tests.Internal/bin/Release/netcoreapp2.1/Abioc.Tests.Internal.dll(.NETCoreApp,Version=v2.1)
Microsoft (R) Test Execution Command Line Tool Version 16.1.0
Copyright (c) Microsoft Corporation.  All rights reserved.
```

**Comment by jayaranigarg** [comment]
@JSkimming : The fix is in process of getting inserted in dotnet sdk. Meanwhile, you can try running the tests in IDE with your test projects pointing to 16.1.1 and test.

**Comment by JSkimming** [comment]
@jayaranigarg My original issue dotnet/core#2775 occurred in the CLI and only then in two hosted CI systems CircleCI and Travis.

Do you know when updated SDKs will be released?

**Comment by tcoreal** [comment]
I can confirm that the issue is resolved with the newest sdk 2.1.701 and Command Line Tool Version (16.1.1)
